app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -389,5 +370,5 @@ fun DaxCta.addCtaToHistory(newCta: String): String {
 
 fun DaxCta.canSendShownPixel(): Boolean {
     val param = onboardingStore.onboardingDialogJourney?.split("-").orEmpty().toMutableList()
-    return !(param.isNotEmpty() && param.last().contains(ctaPixelParam))
+    return !(param.isNotEmpty() && param.any { it.contains(ctaPixelParam) })
```

Comment:
Changed this from last to any because it shouldn't matter where the position of cta in the history, if it exists it should not be sent again. I also updated the test for this.